docs/dev/libvirt-howto.md

```diff
@@ -19,12 +19,14 @@ crw-rw-rw-+ 1 root kvm 10, 232 Oct 31 09:22 /dev/kvm
 If it is missing, try some of the ideas [here][kvm-install].
 
 ### Install and Enable Libvirt
-On Fedora, CentOS/RHEL:
+On Fedora or RHEL systems with QEMU(2.4+):
```

Comment:
I think floor versions like this belong in `docs/dev/dependencies.md` under a new subsection:

```markdown
## Runtime Dependencies

...FIXME Terraform...

### Libvirt

Versions of QEMU less than 2.4 lack support for `-fw_cfg`, which is required for Ignition.
QEMU 2.4 and later may or may not work, we're currently getting testing in place to check at least some versions.
```

Or something very roughly like that.  Thoughts?

Reply:
Sure we can document that there too. But I did wanted to mention that, if somehow your RHEL install does have qemu version 2.4+, the installer may work.